.github/workflows/on-release-tag.yml

```diff
@@ -132,6 +133,7 @@ jobs:
             docker.io/kadalu/kadalu-csi:${{ steps.vars.outputs.tag }}
             docker.io/kadalu/kadalu-csi:latest
           build-args: |
+            builder_version=${{ steps.vars.outputs.tag }}
```

Comment:
Will the `builder_version` ever be different from `version`, if they are always same we can use only `version` here and in subsequent yml/Dockerfiles?

Reply:
not yet sure. Lets keep it this way for a month or 2 and see where we can optimize, and how the other platform story evolves.